config/cache_test.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Update for this year and TestGrid Authors?

Reply:
Same for the other file.

Reply:
Done